features/RST15_audit_events_admin_nfrs.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Admin interface non-functional requirements
+
+    Background:
+	
+	Given fixtures from "bdd_auth" are loaded
```

Comment:
What is bdd_auth?
Not sure this would make sense to the business, maybe we can name it to something more descriptive?